src/components/styled/controlButton.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+const StyledControlButton = styled.div`
```

Comment:
Sorry for the really silly question, but all of the components now reference a StyledButton, but this defined StyledControlButton. Is this needed?

Reply:
This is a good question, and a design thing we can discuss to avoid confusion. 
the const name is internal to the file, default exported. The component importing it can imported it as whatever name it wants.

Reply:
I see. Personally, I feel that the names should be consistent, but we can always do that later.

Reply:
Standardized